InformationScripting/src/graph/PropertyMap.cpp

```diff
@@ -41,4 +47,19 @@ Property InformationScripting::PropertyMap::operator[](const QString& key) const
 	Q_ASSERT(false);
 }
 
+Property& InformationScripting::PropertyMap::operator[](const QString& key)
+{
+	for (auto& content : properties_)
+		if (content.first == key) return content.second;
+	qDebug() << "No object with name" << key;
```

Comment:
I think during development this is suitable, later on we should change it as you suggest.